bin/utils/print-joblog-to-file.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/sh
+#######################################################################
+# This program and the accompanying materials are made available
+# under the terms of the Eclipse Public License v2.0 which
+# accompanies this distribution, and is available at
+# https://www.eclipse.org/legal/epl-v20.html
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Copyright Contributors to the Zowe Project. 2020
+#######################################################################
+
+# Function: Print a JES job log to a USS file
+# Inputs  - jobname and jobid of job to be printed
+# Outputs - a USS file containing the job log, which will be placed in $jobname-$jobid.log
+#         - The job will be purged from the JES spool
+# Uses    - The TSO OUTPUT command
+
+# identify this script
+SCRIPT_DIR="$(dirname $0)"
+SCRIPT="$(basename $0)"
+echo script $SCRIPT started from $SCRIPT_DIR
+
+if [[ $# -ne 2 ]]
+then
+echo; echo $SCRIPT Usage:
+cat <<EndOfUsage
+
+    $SCRIPT jobname jobid
+
+for example
+
+    $SCRIPT myjob   job65262
+    $SCRIPT zwe1sv  stc00406
+
+The output will be placed in $jobname-$jobid.log and the job will be purged.
+EndOfUsage
+echo script $SCRIPT ended 
+exit 
+fi
+
+jobname=$1
+jobid=$2
```

Comment:
I think we want to consider passing in the jobname and id using opts as not doing that has caused problems elsewhere

Reply:
Both parms are mandatory and there are no optional ones, so using opt would make the script and its usage more complicated.  This script is called from `zowe-support.sh` so we don't need to protect against manual errors.